server/sql_migrations/20200602163951_create_maintenance_requests.up.sql

```diff
@@ -1,7 +1 @@
-CREATE TABLE IF NOT EXISTS maintenance_requests
-(
-    ID int,
-    Request varchar,
-    UserSubmitted int,
-    DateSubmitted timestamp
-);
+CREATE TABLE IF NOT EXISTS maintenance_requests (id serial PRIMARY KEY, request varchar, user_submitted int, date_submitted timestamp);
```

Comment:
is there a reason you in-lined everything? I kinda liked it when it's formatted so you can clearly see what's changed.

Reply:
+1 for not inlining.